plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/FileSystemExchange.java

```diff
@@ -67,7 +67,7 @@
         implements Exchange
 {
     private static final Pattern PARTITION_FILE_NAME_PATTERN = Pattern.compile("(\\d+)_(\\d+)\\.data");
-    private static final char[] RANDOMIZED_PREFIX_ALPHABET = "abcdefghijklmnopqrstuvwzyz0123456789".toCharArray();
+    private static final char[] RANDOMIZED_PREFIX_ALPHABET = "abcdef0123456789".toCharArray();
     private static final int RANDOMIZED_PREFIX_LENGTH = 6;
```

Comment:
Do we need to increase this now? This takes us from `2176782336` combinations to only `16777216`, or 0.0077% of the previous.

Reply:
Should these assumptions be persisted as a code comment, documenting why 6 is still sufficient prefix length?